src/gap_to_julia.jl

```diff
@@ -17,12 +17,21 @@
 """
     gap_to_julia_internal(::Type{T}, x::Any, rec_dict::JuliaCacheDict, ::Val{recursive}) where {T, recursive}
 
-returns an object of type `T` that corresponds to the GAP object `x`.
-
-The function `gap_to_julia` may call `gap_to_julia_internal`,
+Return an object of type `T` that corresponds to the GAP object `x`.
+
+The value of `recursive` can be `GAP.BoolVal(true)` or `GAP.BoolVal(false)`,
```

Comment:
`GAP.BoolVal` is an internal helper that only exists as an optimization. I'd not mention it much in the text...
```suggestion
The value of `recursive` can be `Val(true)` or `Val(false)`,
```